components/classes/atm.schema.json

```diff
@@ -0,0 +1,48 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/classes/atm",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "ATM",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:extends": ["https://ns.adobe.com/xdm/data/record"],
+  "description": "For ATM information such as active status, addrerss, ID and name. ",
+  "definitions": {
+    "properties": {
+      "atm": {
```

Comment:
Since you will be removing the nesting root level properties from both the atm and branch classes, please name the properties more semantically as they no longer would be in a context of a outer level filed. The properties in this schema would become
isAtmActive
atmAddress
atmID (Please note all IDs are named as ID and not ID/id)
atmName

The same comment applies to the branch class too.